.github/workflows/run-checker-daily.yml

```diff
@@ -66,6 +66,7 @@ jobs:
           enable-fips,
           enable-fips enable-acvp-tests,
           enable-fips no-tls1_3,
+          enable-fips disable-fips186-4-ffc,
```

Comment:
Could you please add a temporary commit that adds this test run to run-checker-ci.yml? It would be nice to see that this build passes.

Reply:
Running https://github.com/openssl/openssl/actions/runs/11456552021/job/31874955455?pr=25720